Makefile

```diff
@@ -45,6 +45,12 @@ discovery:
 admission-controller:
 	$(GO) -ldflags '$(LDFLAGS)' -o images/tidb-operator/bin/tidb-admission-controller cmd/admission-controller/main.go
 
+backup-manager:
+	$(GO) -ldflags '$(LDFLAGS)' -o images/backup-manager/bin/backup-manager cmd/backup-manager/main.go
```

Comment:
should we add `tidb-` prefix too? see `tidb-admission-controller`

Reply:
Good catch